### DISCUSSION TITLE
Improve captions styling

### DISCUSSION THREAD

**Discussion by Petarkco** [f1multiviewer/issue-tracker#74] [Feature requests] [answered]
The default styling for captions aren't great for quick readability.
![image](https://user-images.githubusercontent.com/25685344/175748939-aca7ccab-ef29-46eb-b96a-697b945f6d20.png)
Suggestion is to make it more like YouTube captions with word wrapping etc

**Comment by f1multiviewer** [comment, marked as answer]
This should be improved in the next release!
